pkg/controllers/nodeclaim/termination/suite_test.go

```diff
@@ -153,10 +158,23 @@ var _ = Describe("Termination", func() {
 		// Expect the node and the nodeClaim to both be gone
 		Expect(env.Client.Delete(ctx, nodeClaim)).To(Succeed())
 		ExpectReconcileSucceeded(ctx, nodeClaimTerminationController, client.ObjectKeyFromObject(nodeClaim)) // triggers the node deletion
+		nodeClaim.StatusConditions().MarkTrue(v1beta1.Terminating)
```

Comment:
Why do we need to set this? should't that happen in the reconciliation loop for this test?